elixir/apps/web/lib/web/live/actors/show.ex

```diff
@@ -450,6 +458,31 @@ defmodule Web.Actors.Show do
       </:content>
     </.section>
 
+    <.section>
+      <:title>Groups</:title>
+
+      <:content>
+        <.live_table
+          id="groups"
+          rows={@groups}
+          row_id={&"group-#{&1.id}"}
+          filters={@filters_by_table_id["groups"]}
+          filter={@filter_form_by_table_id["groups"]}
+          ordered_by={@order_by_table_id["groups"]}
+          metadata={@clients_metadata}
```

Comment:
```suggestion
          metadata={@groups_metadata}
```

Reply:
Whoops, good catch!  Should be fixed now.